elastalert/test_rule.py

```diff
@@ -4,18 +4,31 @@
 from __future__ import print_function
```

Comment:
Note that py3 __future__ imports have weird interactions with non-future printing. In line with the below, should someone attempt to use test_rule as a library, they should be aware of it.